docs/DeveloperGuide.md

```diff
@@ -926,24 +927,41 @@ Use case ends.
 
 **MSS**
 
-1. User wants to assign an alias name for a command.
+1. User requests to assign an alias name for a command.
 2. SunRez validates the alias name and command.
 3. SunRez saves the alias mapping to the user’s preferences.
 
 Use case ends.
 
 **Extensions**
 
-* 1a. Alias name is invalid.
-    * 1a1. SunRez shows an error message. 
+* 2a. Alias name is invalid.
+    * 2a1. SunRez shows an error message. 
 
         Use case ends.
 
-* 2a. An alias with the same name exists.
-    * 2a1. SunRez overwrites the existing alias. 
+* 3a. An alias with the same name exists.
+    * 3a1. SunRez overwrites the existing alias. 
 
         Use case resumes from step 3.
```

Comment:
Should this be `Use case ends.` also? It seems a bit weird that it branches from 3 then goes back to 3.